src/inmanta/server/config.py

```diff
@@ -105,17 +106,15 @@ def get_bind_port() -> int:
     if Config.is_set("server", "bind-port") or Config.is_set("server", "bind-address"):
         # Use new bind-port option
         if Config.is_set("server_rest_transport", "port"):
-            warnings.warn(
+            inmanta.warnings.warn(
```

Comment:
Is there a particular reason to drop the information that this is a deprecation? One of the benefits of warnings is that you have more information than just "WARNING". You can use [warning filter](https://docs.python.org/3/library/warnings.html#warning-filter) to control which ones you want to see.

With pytest for example you can ask explicitly: only show me deprecation warnings: https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html

Reply:
The reason is that we ignore all warnings by default: https://github.com/inmanta/inmanta-core/blob/0aba1b03937bad3900fc46673726e14e1124e81b/src/inmanta/warnings.py#L121 in order not to get DeprecationWarnings from third-party libraries. But I see we can also define a filter on module. @sanderr I see you implemented these warning filters in https://github.com/inmanta/inmanta-core/pull/1928. Do you remember why you didn't filter on module name?

Reply:
Honestly it might just have been an oversight. I remember back when I implemented this I was pretty unexperienced with warnings. Filtering on module name would make more sense in hindsight, and we could then probably even get rid of the `InmantaWarning` class.